sdk/storage/inc/datalake/file_client.hpp

```diff
@@ -52,10 +51,24 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     std::string Date;
     std::string RequestId;
     std::string Version;
+    Azure::Core::Nullable<std::string> ClientRequestId;
+  };
+
+  struct FileDownloadInfo
+  {
+    std::string ETag;
+    std::string LastModified;
+    int64_t ContentLength = 0;
+    DataLakeHttpHeaders HttpHeaders;
+    std::map<std::string, std::string> Metadata;
+    Azure::Core::Nullable<bool> ServerEncrypted;
+    Azure::Core::Nullable<std::string> EncryptionKeySHA256;
   };
 
   using FileInfo = PathInfo;
   using FileCreateOptions = PathCreateOptions;
+  using FileContentInfo = Blobs::BlobContentInfo;
+  using DownloadFileOptions = Blobs::DownloadBlobToBufferOptions;
```

Comment:
This should be moved to options.hpp

Reply:
I was trying to avoid including blob headers in `datalake_options.hpp`. Is it really necessary to move it in there?

Reply:
what's the point of avoiding including blob headers in `datalake_options.hpp`?

Reply:
Edit: moved the typedefs in `datalake_options.hpp` and `datalake_responses.hpp`.